guide/14-deep-learning/how_sr3_works.ipynb

```diff
@@ -0,0 +1,234 @@
+{
```

Comment:
<p>If possible, add hyper links to explain the concepts of diffusion model, denoising diffusion probabilistic model, stochastic denoising process. </p><p>You can also add hyperlinks to U-Net model. </p><p>Add hyper links to forward markovian diffusion process also. </p><p> </p><p>These hyperlinks will help users who are not aware of these terms and would want to understand it further. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1664/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>